frontend/lib/App.dart

```diff
@@ -1,10 +1,10 @@
 import 'package:adpv_frontend/Repository/AbstractEndpointRepository.dart';
-import 'package:adpv_frontend/Repository/MockRepository.dart';
 import 'package:flutter/material.dart';
 
-import 'CompareEndpoints/CompareEndpointsView.dart';
-import 'EndpointList/EndpointNavigator.dart';
-import 'Profile/ProfileView.dart';
+import 'Routing/EndpointNavigator.dart';
+import 'Views/CompareEndpointsView.dart';
+import 'Views/ProfileView.dart';
+
 
 const String endpointList = "Endpoint List";
 const String compareEnpoints = "Compare";
```

Comment:
to jest komentarz troche nie tutaj, ale zauwazylam ze w wielu miejscach zamiast Endpoints mamy Enpoints, więc jakbyscie gdzies na to trafili, to mozecie przy okazji zmienic